packages/fluent-editor/src/mathlive/formats.ts

```
@@ -0,0 +1,74 @@
+  initFlag = false
+  mode: MathliveBlotMode
+  constructor(scroll: Root, domNode: Node) {
+    super(scroll, domNode)
+    const dom = domNode as HTMLElement
+    dom.setAttribute('contenteditable', 'false')
+    this.mode = (dom.getAttribute('mode')
+      || 'only-read') as MathliveBlotMode
+  }
+
+  static create(obj: { value: string, mode: MathliveBlotMode }) {
+    const el = super.create() as MathfieldElement
```

Biome

[error] Using super in a static context can be confusing.

super refers to a parent class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)